### ISSUE TITLE
Add Facebook Login

### ISSUE THREAD

**Issue by alistairholmes** [alistairholmes/devjournal#3]
Using Firebase Auth add facebook login authentication to the GoogleSignIn activity. Please **do not** remove google authentication.

References: 
- https://firebase.google.com/docs/auth/android/facebook-login
- https://github.com/firebase/quickstart-android/tree/master/auth#facebook-login-setup

Please follow Contribution guidelines in README.md

**Comment by ThinkHattke** [comment]
Can I try this one?

**Comment by codetillamgone** [comment]
Hello, I want to take this on

**Comment by alistairholmes** [comment]
@ThinkHattke have you made any progress on this issue? If not @codetillamgone can work on it.

**Comment by ThinkHattke** [comment]
Yes, I have just started, but I guess @codetillamgone can take this one. I will try something else then.
